Chatify/chat/redis.py

```diff
@@ -0,0 +1,13 @@
+from .websocket_utils import send_chat_message
+import redis
+from django.utils import timezone
+import pytz
+
+
+def check_last_login(user_id):
+    r = redis.Redis()
+    last_login = r.get(f"user:{user_id}:last_login")
```

Comment:
user_last_active_time_key_identifier = f"user:{user_id}:last_login"

Reply:
can we write this big user_last_active_time_key_identifier variable name ?